test/logger.test.js

```diff
@@ -226,6 +269,56 @@ test('The logger should accept a custom genReqId function', t => {
   })
 })
 
+t.test('The logger should accept custom serializer', t => {
+  t.plan(9)
+  var fastify = null
+  var stream = split(JSON.parse)
+  try {
+    fastify = Fastify({
+      logger: {
+        stream: stream,
+        level: 'info',
+        serializers: {
+          req: function (req) {
+            return {
+              url: req.url
+            }
+          }
+        }
+      }
+    })
+  } catch (e) {
+    t.fail()
```

Comment:
which case `Fastify` function throws an error?

Reply:
Just shamefully copy paste another test :/ I will remove that!